src/indigo/less/inline-edit-text-input.less

```diff
@@ -0,0 +1,77 @@
+.inline-edit-text-input {
```

Comment:
Fajn ze jsi dodrzel nazvoslovi napric celou komponentou.
Jenom nevim jestli by se to nedalo jeste zkratit na napr: 'inline-edit-input'.
Nepouzivame zadnou metodiku, tak se v tom nesmime uplne rozvasnit.

Reply:
ostal by som pri `inline-edit-input` ... dalsia potom bude `inline-edit-textarea` a pod.